packages/svelte-ux/src/lib/components/QuickSearch.svelte

```diff
@@ -62,12 +64,15 @@
   }}
 >
   <div class="overflow-auto max-h-[min(90dvh,600px)] min-w-[400px] py-1">
-    <SelectList
+    <SelectField
       icon={mdiMagnify}
       placeholder="Search..."
+      showToggleIcon={false}
```

Comment:
maybe invert this an call it `hideToggleIcon`.  I like boolean props to always default to `false` and not require a value.

Reply:
Made this change.